lib/addons/p5.dom.js

```diff
@@ -550,12 +554,19 @@
    * }
    * </code></div>
    */
+
```

Comment:
There should be an additional method signature identifying the option to pass in a DOM element rather than a boolean. https://github.com/processing/p5.js/wiki/Inline-documentation#additional-signatures